functions/api/image/nfts/asset/[[index]].tsx

```diff
@@ -2,6 +2,7 @@
 import { ImageResponse } from '@vercel/og'
 import React from 'react'
 
+import { blocklistedCollections } from '../../../../../src/nft/utils/collection'
```

Comment:
collection.ts imports types using a paths shortcut (`nft/types`) so this causes typecheck to fail.

rather than go through the tree and convert things to use relative paths, I think it'd be easiest to break `blocklistedCollections` into its own file, `../../../../../src/nft/utils/blocklist.ts`, and avoid importing into that file. that way _this_ import can stay clean and pass typecheck.